src/request/export/Export.css

```diff
@@ -42,25 +42,21 @@ ul.nq-list li > .nq-icon {
     padding-bottom: 0;
 }
 
-.page#recovery-words .nq-text {
-    text-align: center;
-    color: var(--nimiq-orange); /* Can be removed when CSS specificity is fixed in nimiq-styles */
-    margin-bottom: 0;
-}
-
 .page#recovery-words .recovery-words {
-    max-height: 100%;
+    height: 39rem;
     margin: 0;
     position: relative;
-    margin: 0 -1.5rem -3rem;
-    padding: 3rem 0;
-    -webkit-mask-image: linear-gradient(0deg , rgba(255,255,255,0), rgba(255,255,255, 1) 20%, rgba(255,255,255,1) 80%, rgba(255,255,255,0));
-    mask-image: linear-gradient(0deg , rgba(255,255,255,0), rgba(255,255,255, 1) 20%, rgba(255,255,255,1) 80%, rgba(255,255,255,0));
+    margin: -4rem -4rem -5rem;
+    padding: 4rem 3rem;
```

Comment:
```suggestion
    padding: 2rem 3rem;
```

Reply:
When reducing the padding, the gradient would overlap the words, which was explicitly requested not to happen by Julian. We would need to reduce the gradient to 2rem, which is too strong I think.